src/command/root-command.ts

```diff
@@ -12,17 +14,31 @@ export class RootCommand {
   @ExternalOption('bee-api-url')
   public beeApiUrl!: string
 
+  @ExternalOption('config-folder')
+  public configFolder!: string
+
   // CLASS FIELDS
 
+  public appName = 'swarm-cli'
+
   public bee!: Bee
 
+  public commandConfig!: CommandConfig
+
   /**
    * Init Root command fields
    *
    * if BEE_API_URL environment variable has been set the CLI will use that connection string
    */
   protected init(): void {
-    this.beeApiUrl = this.beeApiUrl === defaultBeeApiUrl ? process.env.BEE_API_URL || this.beeApiUrl : this.beeApiUrl
+    this.commandConfig = new CommandConfig(this.appName, this.configFolder)
+    this.beeApiUrl = this.optionPassed(beeApiUrl)
```

Comment:
What's the point of assigning the value to itself (`this.beeApiUrl = this.beeApiUrl`)?
Both branches fall back to this. Maybe this can be simplified.

Reply:
Thanks for noticing, I change this conditional statement to a good old `if` one.

Also rearranged the priority in that way that I mentioned before